tests/scripts/single_card/nightly/run_common_models.sh

```diff
@@ -9,7 +9,7 @@ fi
 fail=0
 
 echo "Running common models for archs"
-
+env pytest models/experimental/functional_mobilenetv2/test/test_ttnn_mobilenetv2.py ; fail+=$?
```

Comment:
Please add this as a symlink in `tests/nightly/single_card/`

Look at the other models in that folder for reference